experimental/origin-graphql/src/typeDefs/Growth.js

```diff
@@ -145,10 +145,11 @@ module.exports =
   type Query {
     # first property specifies the number of items to return
     # after is the cursor
-    campaigns(first: Int, after: String, walletAddress: ID!): GrowthCampaignConnection
-    campaign(id: String, walletAddress: ID!): GrowthCampaign
-    isEligible: EligibilityInfo
+    campaigns(first: Int, after: String): GrowthCampaignConnection
+    campaign(id: String): GrowthCampaign
+    invites(walletAddress: ID!, campaignId: Int!): ReferralAction
```

Comment:
I think it would be better to return this inside GrowthCampaigns's actions list

Reply:
Ah sorry, yes you are right. this should be returned as an Action from the campaign endpoint. will update soon...